internal/iam/command.go

```diff
@@ -32,8 +32,8 @@ func New(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Command {
 	cmd.AddCommand(newPoolCommand(prerunner))
 	cmd.AddCommand(newProviderCommand(prerunner))
 	cmd.AddCommand(newRBACCommand(cfg, prerunner))
-	cmd.AddCommand(newIPFilterCommand(prerunner))
-	cmd.AddCommand(newIPGroupCommand(prerunner))
+	cmd.AddCommand(newIpFilterCommand(prerunner))
```

Comment:
Any reason why we're switching back to camel case? I thought you linked to a Golang style guide that said this should look like "IPFilter"

Reply:
I originally thought it should be `IPFilter`, however all the autogenerated methods in our SDK use the style of `Ip`, so to be consistant with our public SDK, I'm matching the camelcase